packages/block-editor/src/components/block-preview/index.js

```diff
@@ -28,12 +28,22 @@ function BlockPreview( props ) {
 	);
 }
 
-export function BlockPreviewContent( { name, attributes, innerBlocks, settings } ) {
-	const block = createBlock( name, attributes, innerBlocks );
+export function BlockPreviewContent( { blocks, settings } ) {
+	if ( ! blocks ) {
+		return null;
+	}
+
+	const theBlocks = Array.isArray( blocks ) ? blocks : [ blocks ];
```

Comment:
Let's use [`castArray` from lodash](https://lodash.com/docs/4.17.14#castArray). It's already being used in other places in Gutenberg that are able to accept a block or a list of blocks.